src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/GRBReviewFormStepWrapper/index.tsx

```diff
@@ -88,43 +76,62 @@ function GRBReviewFormStepWrapper<
   const nextStep: StepHeaderStepProps | undefined =
     currentStepIndex > -1 ? steps[currentStepIndex + 1] : undefined;
 
-  /**
-   * Submits step if fields are valid
-   *
-   * If invalid, redirects to GRB Review tab or `path` prop with no error messages
-   */
+  /** Submits form step and redirects user */
   const submitStep = useCallback(
-    async (path?: string) => {
-      // Don't submit if invalid or no changes
-      if (!isValid || !isDirty) {
-        return history.push(`${grbReviewPath}/${path || ''}`);
+    async ({
+      shouldValidate = true,
+      path = ''
+    }: {
+      /** If false, does not submit if fields are invalid. Defaults to true. */
+      shouldValidate?: boolean;
+      /** Appends to `grbReviewPath` - leave undefined to return to GRB review tab after submit */
+      path?: string;
+    }) => {
+      // Redirect user without submit if no changes or skipping validation
+      if (!isDirty || (!shouldValidate && !isValid)) {
+        return history.push(`${grbReviewPath}/${path}`);
       }
 
-      return handleSubmit?.(values =>
-        onSubmit?.({ systemIntakeID: systemId, ...values }).then(() =>
-          history.push(`${grbReviewPath}/${path || ''}`)
-        )
+      return handleSubmit(values =>
+        onSubmit({ systemIntakeID: systemId, ...values })
+          .then(() => history.push(`${grbReviewPath}/${path}`))
+          // Show error message if validating fields, otherwise redirect
+          .catch(() => {
+            if (shouldValidate) {
+              showMessage(t('setUpGrbReviewForm.error'), { type: 'error' });
+            } else {
+              history.push(`${grbReviewPath}/${path}`);
+            }
+          })
       )();
     },
-    [grbReviewPath, isValid, isDirty, handleSubmit, onSubmit, history, systemId]
+    [
+      grbReviewPath,
+      isValid,
+      isDirty,
+      handleSubmit,
+      onSubmit,
+      history,
+      systemId,
+      showMessage,
+      t
+    ]
   );
 
   /**
    * Formats form steps for stepped header
    */
   const formatSteps = useCallback(async () => {
-    const { grbReviewType, grbReviewers, grbPresentationLinks } = grbReview;
+    const { grbReviewType, grbReviewers, grbDate } = grbReview;
 
     // Validate form steps with Yup
 
-    const reviewTypeIsValid =
-      await GrbReviewFormSchema.grbReviewType.isValid(grbReviewType);
+    const reviewTypeIsValid = await GrbReviewFormSchema.reviewType.isValid({
+      grbReviewType
+    });
 
     const presentationIsValid = await GrbReviewFormSchema.presentation.isValid({
-      recordingLink: grbPresentationLinks?.recordingLink,
-      presentationDeckFileData: grbPresentationLinks?.presentationDeckFileName
-        ? {}
-        : undefined
+      grbDate
```

Comment:
Since these are using the persisted data from `grbReview` and not the live form data - and the `Next` buttons are relying on this validation to progress, the only way to progress through forms, it to wait for the autosave to trigger, then the next button becomes disabled.